File: eTeatar/eTeatar.Model/Repertoar.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTeatar.Model
{
    public class Repertoar
    {
        public int RepertoarId { get; set; }

        public string Naziv { get; set; }

        public string Opis { get; set; } 

        public DateTime DatumPocetka { get; set; }

        public DateTime DatumKraja { get; set; }
    }
}

File: eTeatar/UI/eteatar_desktop/lib/models/repertoar.dart
import 'package:json_annotation/json_annotation.dart';

part 'repertoar.g.dart';

@JsonSerializable()
class Repertoar {
  int? repertoarId;
  String? naziv;
  String? opis;
  DateTime? datumPocetka;
  DateTime? datumKraja;
  
  Repertoar(
    this.repertoarId,
    this.naziv
  );

  factory Repertoar.fromJson(Map<String, dynamic> json) => _$RepertoarFromJson(json);

  /// Connect the generated [_$GlumacToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$RepertoarToJson(this);
}